### PR TITLE
liblc3 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

- https://github.com/google/liblc3/pull/53
- https://github.com/google/liblc3/pull/54

### CODE REVIEW

**Review by SMillerDev — CHANGES_REQUESTED** [review]
Let's wait for the next release when it passes the audit

**Comment by chenrui333** [comment]
> Let's wait for the next release when it passes the audit

sounds good, convert to draft for now.

File: .github/autobump.txt
Comment (carlocab) [inline .github/autobump.txt:1098]:
```suggestion
liblc3
```

File: Formula/lib/liblc3.rb
Comment (carlocab) [inline Formula/lib/liblc3.rb:24]:
```suggestion
    ENV.append "LDFLAGS", "-Wl,-rpath,#{rpath}"
    system "meson", "setup", "build", *args, *std_meson_args
```
This should likely allow you to get rid of the patch (which upstream doesn't seem so keen on anyway).

**Review by carlocab — DISMISSED** [review]

File: Formula/lib/liblc3.rb
Comment (chenrui333) [reply Formula/lib/liblc3.rb:21]: that does not quite work, needs to be part of the meson build

```
  Files with missing rpath:
    /usr/local/Cellar/liblc3/1.1.0/bin/dlc3
    /usr/local/Cellar/liblc3/1.1.0/bin/elc3
```

Comment (carlocab) [reply Formula/lib/liblc3.rb:20]: Hm, odd. Let me look into it.

**Review by MikeMcQuaid — APPROVED** [review]
Looks good to me, thanks @chenrui333. Let @carlocab and/or @SMillerDev 👍🏻 before merging.

**Review by p-linnane — APPROVED** [review]

**Comment by chenrui333** [comment]
gentle ping @SMillerDev @carlocab 🙏

File: Formula/lib/liblc3.rb
Comment (SMillerDev) [inline Formula/lib/liblc3.rb:14]: Since they seemed hesitant to merge it, it might be better to create an issue with the problem to link to?

**Review by carlocab — APPROVED** [review]
Ok with me for now, I've followed up the RPATH issue upstream at https://github.com/google/liblc3/pull/53.

**Review by SMillerDev — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Pull request merged**
